cve_bin_tool/cli.py

```diff
@@ -81,6 +81,11 @@ def __call__(self, parser, namespace, values, option_string=None):
 
 def main(argv=None):
     """Scan a binary file for certain open source libraries that may have CVEs"""
+    if sys.version_info < (3, 7):
+        raise OSError(
+            "Python no longer provides security updates for version 3.6 as of December 2021. Please upgrade to python 3.7+ to use CVE Binary Tool."
+        )
+        exit()
```

Comment:
I think you can actually leave off the exit() here because the raised error is going to halt execution, but since it's harmless I'm not going to bother to make you change it.